crd-reference/check.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+
+# Checks to make sure that the reference docs in the content/en/docs/reference directory
+# are up-to-date with the Spin Operator CRDs
+
+script_dir=$(dirname "$0")
```

Comment:
`script_dir` isn't used. I would follow the pattern in `generate.sh` using a `root_dir` and change the relative paths below to use it.

Reply:
`script_dir` is being used to cd into that directory.